db-scripts/createdb.sql

```diff
@@ -91,7 +91,7 @@ create table if not exists membership_requests(
    user_id integer not null,
    PRIMARY KEY(id),
    FOREIGN KEY(plan_id) REFERENCES membership_plan(id) on update cascade,
-   FOREIGN KEY(user_id) REFERENCES users(id) on update cascade
+   FOREIGN KEY(user_id) REFERENCES users(id) on update cascade on delete cascade
```

Comment:
can we delete or comment out all membership related tables

Reply:
deleted